src/overlays/actors/ovl_En_Zod/z_en_zod.c

```diff
@@ -77,12 +68,23 @@ static ColliderCylinderInit sCylinderInit = {
     { 60, 40, 0, { 0, 0, 0 } },
 };
 
-static AnimationHeader* sAnimations[] = {
-    &gTijoPlayingVivaceAnim, &gTijoReadyToPlayAnim,      &gTijoArmsFoldedAnim,
-    &gTijoPlayingLentoAnim,  &gTijoPlayingAndantinoAnim,
-};
+typedef enum EnZodAnimation {
+    /* -1 */ ENZOD_ANIM_NONE = -1,
+    /* 0  */ ENZOD_ANIM_PLAYING_VIVACE,
+    /* 1  */ ENZOD_ANIM_READY_TO_PLAY,
+    /* 2  */ ENZOD_ANIM_ARMS_FOLDED,
+    /* 3  */ ENZOD_ANIM_PLAYING_LENTO,
+    /* 4  */ ENZOD_ANIM_PLAYING_ANDANTINO,
+    /* 5  */ ENZOD_ANIM_MAX
```

Comment:
I may be wrong, but I think we were right-aligning those comments, right?
```suggestion
    /* -1 */ ENZOD_ANIM_NONE = -1,
    /*  0 */ ENZOD_ANIM_PLAYING_VIVACE,
    /*  1 */ ENZOD_ANIM_READY_TO_PLAY,
    /*  2 */ ENZOD_ANIM_ARMS_FOLDED,
    /*  3 */ ENZOD_ANIM_PLAYING_LENTO,
    /*  4 */ ENZOD_ANIM_PLAYING_ANDANTINO,
    /*  5 */ ENZOD_ANIM_MAX
```